PolicyAndKeys-Client/Program.cs

```diff
@@ -189,11 +206,15 @@ private static string[] ProcessParametersInput()
                     Console.WriteLine($"For Command: {cmdType.ToString()} specify path of {resType.ToString()} ");
                     break;
                 case CommandType.UPDATE:
+                case CommandType.UPLOADSECRET:
+
+                    Console.WriteLine($"For Command: {cmdType.ToString()} (space separated) specify Id and path of {resType.ToString()} ");
+                    if (resType != ResourceType.policies) Console.WriteLine("optionally, type test for both, if you want to simply try this out");
+                    break;
+
                 case CommandType.UPLOADCERTIFICATE:

```

Comment:
Can we make sure that GenerateKey, uploadcert, uploadpkcs12, uploadcert, getactive key dont work for policies.

Reply:
yes they wont, because the resource type is different